app/services/court_hearings/create_in_nomis.rb

```diff
@@ -1,7 +1,7 @@
 module CourtHearings
   class CreateInNomis
     def self.call(move, court_hearings)
-      booking_id = move.profile.latest_nomis_booking_id
+      booking_id = move.profile.person.latest_nomis_booking_id
```

Comment:
We can do this later, up to you:
`move.profile.person` is used in many places now, so we should remove the 

```
  def person
    raise 'Attempt to Access to person!!!'
  end
```
on the Move and allow to access the person through the profile.

Not high priority!

Reply:
We've agreed to do this in a separate PR.